### PR TITLE
[temp.local] Remove space between angle brackets in example

### PR DESCRIPTION
**Pull request by sdkrystian**
We don't do this very much, if at all.

### CODE REVIEW

File: source/templates.tex [hunk 1]
```diff
@@ -4747,7 +4747,7 @@
 };
 
 template<class T, template<class> class U = T::template Base> struct Third { };
-Third<Derived<int> > t;                 // OK: default argument uses injected-class-name as a template
+Third<Derived<int>> t;                  // OK: default argument uses injected-class-name as a template
```
Comment (jensmaurer) [inline source/templates.tex:4750]: I think we don't want to change this under the "core sections show a variety of syntactically valid styles" doctrine.

Comment (sdkrystian) [reply source/templates.tex:4750]: @jensmaurer Could we at least make the spacing symmetric? :)

Comment (sdkrystian) [reply source/templates.tex:4750]: Ah, the one place I didn't look when checking for more instances was... [temp]. I can still wait for @zygoloid to look this over if we want, but I agree with you that we shouldn't make the change

**Comment by jensmaurer** [comment]
Author agreed not to apply the change.

**Pull request closed without merging**